how_to_ask_for_help.html

```diff
@@ -1,46 +1,78 @@
+                <div class="navbar-brand">
+                    <a class="navbar-item" href="index.html"><img src="assets/img/codecademy_logo.png" alt="The codecademy logo" class="logo"></a>
+                    <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbar-items">
+                        <span aria-hidden="true"></span>
+                        <span aria-hidden="true"></span>
+                        <span aria-hidden="true"></span>
+                    </a>
+                </div>
+                <div class="navbar-menu" id="navbar-items">
+                    <div class="navbar-end">
+                        <a class="navbar-item" href="./index.html">Home</a>
+                        <a class="navbar-item" href="./staff.html">Server Staff</a>
+                        <a class="navbar-item" href="./contact.html">Contact Us</a>
+                        <a class="navbar-item" href="./sharing_code.html">Sharing Code</a>
+                    </div> 
+                </div>
+            </nav>
+        </header>
+        <main class="container has-text-centered">
+            <h1 class="is-size-1">How to get Coding Help</h1>  
+            <p class="p-3">This is a guide on how to get help when you run into a problem while coding.</p>
+            <p class="p-3">Most people are very keen to offer help if they can, but we have noticed a few problems people make that mean it is harder for others to help them. This might cause you to get frustrated, and not get the help you need.</p>
+            <article class="container content has-background-white p-6 mt-4">
+                    <section>
+                        <h2 class="is-size-2">1. Let us see your code</h2>
+                        <p>We probably cannot help you with your problem if we cannot see what's wrong! And sometimes it is not immediately clear what is wrong. We may need to test and edit the code ourselves to diagnose and find a solution to the problem.</p>
+                        <ul>
+                            <li>Put the code in a format we can edit. Screenshots or photos of the screen will not work for this, as people would have to type out all of your code just to start helping.</li>
+                            <li>Include all of your code. If you just put part of the code that you think is the problem, we cannot help if it turns out the problem is in another part of your code. This is a big issue in CSS, where other parts of your CSS file can change things everywhere.</li>
+                            <li>If you change something, post the updated code. This helps people see if you have made error when you have made corrections.</li>
+                            <li>Use sites like <a href="https://github.com/">Github</a> or <a href="https://jsfiddle.net/">jsFiddle</a> to post your code in a way that people can see and work with it.</li>
+                        </ul>
```

Comment:
It might be better to have this section align to the left rather than the centre so that there isn't varying amounts of space between the bullets and the sentences.